### ISSUE TITLE
`prospector` requires too tight a constraint: `pycodestyle==2.0.0`

### ISSUE THREAD

**Issue by pmatos** [PyCQA/prospector#224]
Hi,

I tried installing `prospector` with anything using pip, which requires `pycodestyle==2.0.0`.

Dependencies found by `pip-compile` from `pip-tools`:
```
prospector[with_everything]==0.12.7 requires dodgy>=0.1.9, frosted>=1.4.1, mccabe>=0.5.0, pep8-naming>=0.3.3, pycodestyle==2.0.0, pydocstyle>=2.0.0, pyflakes>=0.8.1, pylint-celery>=0.3, pylint-common>=0.2.5, pylint-django>=0.7.2, pylint-flask>=0.3, pylint-plugin-utils>=0.2.6, pylint>=1.5.6, pyroma==2.0.2, pyyaml, requirements-detector>=0.4.1, setoptconf>=0.2.0, vulture>=0.6
```

However, I have other dependencies that requires looser but incompatible versions like:
```
autopep8==1.3.2           requires pycodestyle>=2.3
```

Is it possible to make the version required by `prospector` less restrictive?

**Comment by peterdemin** [comment]
I had to remove `prospector` from my project because it breaks flake8, which depends on `pycodestyle` of >=2.3.0.

**Comment by dkfellows** [comment]
The stupid thing is that prospector appears to work fine with pycodestyle 2.3.1

**Comment by chocoelho** [comment]
This pinning was fixed on version 0.12.9. Closing this issue since it's fixed.
